tests/Tests/600__ai_apps/680__starburst/681__managed_starburst_monitoring.robot

```diff
@@ -0,0 +1,58 @@
+                ...  container_fs_usage_bytes  container_network_transmit_bytes_total  kube_pod_container_resource_requests    container_memory_usage_bytes
+                ...  container_network_receive_bytes_total  kube_deployment_status_replicas_available  kube_node_status_capacity
+                ...  kube_deployment_status_replicas_unavailable  kube_persistentvolumeclaim_status_phase  kube_pod_container_resource_limits
+                ...  node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate  cluster:namespace:pod_cpu:active:kube_pod_container_resource_limits
+                ...  container_network_receive_packets_total  container_network_transmit_packets_total  kube_running_pod_ready
+                ...  container_cpu_usage_seconds_total  kube_pod_container_status_restarts_total  kube_pod_status_phase  cluster:namespace:pod_memory:active:kube_pod_container_resource_requests
+                ...  jmx_config_reload_success_total  jmx_scrape_duration_seconds  jmx_scrape_cached_beans  jmx_scrape_error
+                ...  jmx_exporter_build_info  jmx_config_reload_failure_total
+                ...  jmx_config_reload_failure_created  jmx_config_reload_success_created  jvm_threads_current  jvm_threads_daemon  jvm_threads_peak
+                ...  jvm_threads_started_total  jvm_threads_deadlocked  jvm_threads_deadlocked_monitor  jvm_threads_state  jvm_buffer_pool_used_bytes
+                ...  jvm_buffer_pool_capacity_bytes  jvm_buffer_pool_used_buffers  jvm_info  jvm_heap_memory_used  jvm_heap_memory_commited
+                ...  jvm_memory_pool_allocated_bytes_total  jvm_memory_bytes_used  jvm_memory_bytes_committed  jvm_memory_bytes_max  jvm_memory_bytes_init
+                ...  jvm_memory_pool_bytes_used  jvm_memory_pool_bytes_committed  jvm_memory_pool_bytes_max  jvm_memory_pool_bytes_init  jvm_classes_loaded
+                ...  jvm_classes_loaded_total  jvm_classes_unloaded_total  jvm_gc_collection_seconds_count  jvm_gc_collection_seconds_sum
+                ...  jvm_memory_pool_allocated_bytes_created  failed_queries  jvm_heap_memory_commited  jvm_heap_memory_used  thread_count  trino_active_nodes
+                ...  trino_free_memory  queries_killed_oom  trino_active_queries  trino_queries_started  trino_queries_failed_external  trino_queries_failed_internal
+                ...  trino_queries_failed_all  trino_failed_queries_user  trino_execution_latency  trino_input_data_rate  input_data_bytes  input_rows
+                ...  cluster_memory_bytes  tasks_killed_oom
+
+
+*** Test Cases ***
+Run STARBURST Query And Check Values Are Not Empty
+    [Documentation]    Verifies the Observatorium metrics values are not none
+    [Tags]    MISV-94
+    ${SSO_TOKEN}    Prometheus.Get Observatorium Token
+    @{value}=    Create List
+    FOR  ${query}   IN   @{serh_querys}
+        ${obs_query_op}=    Prometheus.Run Query    ${STARBURST.OBS_URL}    ${SSO_TOKEN}
```

Comment:
does this kw show sensitive values in the HTML report? Maybe the `${SSO_TOKEN}`

Reply:
even its  showing its expiring in less then 1sec